main/src/ca/uwaterloo/flix/Main.scala

```diff
@@ -623,10 +622,6 @@ object Main {
       opt[Seq[Subeffecting]]("Xsubeffecting").action((subeffectings, c) => c.copy(xsubeffecting = subeffectings.toSet)).
         text("[experimental] enables sub-effecting in select places")
 
-      // Xchaos-monkey
-      opt[Unit]("Xchaos-monkey").action((_, c) => c.copy(xchaosMonkey = true)).
```

Comment:
The chaos monkey can only be enabled internally. It serves no purpose to expose it.